Preconfigure and document GitHub OAuth login (#9917)
camptocamp/c2cgeoportal: doc/integrator/tilegeneration.rst

~~~~~~~~~~~~~

It is possible to run tile generation commands through a web interface located at URL

> ``<application main URL>/tiles/admin/``. This interface is protected by a password

Reviewer: ```suggestion
``<application main URL>/tiles/admin/``. This interface require an authentication
```
